### ISSUE TITLE
Focus Ring Preset Not Working in PrimeVue 4.0.0-beta.3

### ISSUE THREAD

**Issue by beijixiaohu** [primefaces/primevue#5859]
### Describe the bug

Hi,

I followed the instructions on the [official site](https://v4.primevue.org/theming/styled/) to change the default focusRing, but I'm not getting the expected results. For example, when I focus on an input field, I don't see a wider ring around it. I’m a bit confused and not sure if I misunderstood something or if there’s an actual issue. Any help would be appreciated. Thanks a lot!

### Reproducer

https://stackblitz.com/edit/vitejs-vite-54ggtx?file=src%2Fmain.ts

### PrimeVue version

4.0.0-beta.3

### Vue version

3.x

### Language

TypeScript

### Build / Runtime

Vite

### Browser(s)

Chrome 125.0.6422.142

### Steps to reproduce the behavior

1. Click on the input box.

### Expected behavior

A focus ring should be visible.

**Comment by beijixiaohu** [comment]
Oh, looks like I misunderstood something. Got it now, I'll close it.

**Comment by JohnCampionJr** [comment]
Any chance you can provide a tip?  I can't seem to get this working with v4 either.

**Comment by beijixiaohu** [comment]
> Any chance you can provide a tip? I can't seem to get this working with v4 either.

Oops, my bad. I can't really help you out with this one since I've totally forgotten why I closed the issue in the first place. 😅

I just tried it again, and I couldn't get it to work either...

Actually, because my project needed more complex and detailed theme control, I ended up designing my own theme system instead of using PrimeVue's.

If you're still stuck on this, maybe try opening a new issue to get some help. Good luck!
